saml-core/src/test/java/org/keycloak/saml/processing/core/parsers/saml/SAMLAttributeQueryParserTest.java

```diff
@@ -43,7 +43,7 @@ public void initParser() {
         this.parser = new SAMLParser();
     }
 
-    @Test(timeout = 2000000)
+    @Test(timeout = 2000)
```

Comment:
@ahus1 It seems this was changed from `2000` to `2000000` for some reason. @hmlnarik Any chance you remember why it was increased in [this PR](https://github.com/keycloak/keycloak/pull/5017/files#diff-d7d2918cfbcc38ffaae90bea3d00ca904a1b5461098ed60bd5ba125f11de7761R46)?

Reply:
IIRC this was because of debugging and then left in the commit. No harm in keeping it back at 2000 I believe